### PR TITLE
./github/workflows/conflict_reminder.yaml: fix assignee object

### PR DESCRIPTION
**Pull request by lsfreitas**
References the login property of object assignee

### CODE REVIEW

**Review by yaronkaikov — APPROVED** [review]
LGTM, @lsfreitas thanks for the quick fix

**Comment by yaronkaikov** [comment]
@scylladb/scylla-maint please merge this

**Review by Annamikhlin — APPROVED** [review]

File: .github/workflows/conflict_reminder.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ name: Notify PR Authors of Conflicts
 
 on:
   schedule:
-    - cron: '0 10 * * 1,4'  # Runs every Monday and Thursday at 10:00am
-  workflow_dispatch:      # Manual trigger for testing
+    - cron: "0 10 * * 1,4" # Runs every Monday and Thursday at 10:00am
+  workflow_dispatch: # Manual trigger for testing
```
Comment (denesb) [inline .github/workflows/conflict_reminder.yaml:6]: Unrelated changes, please refrain from doing these in the future.

Also, please prepend: `.github:` to the commit subject, so we know which subsystem the commit touches.

Comment (yaronkaikov) [reply .github/workflows/conflict_reminder.yaml]: Removed and edit the PR title and commit

**Review by denesb — APPROVED** [review]

**Pull request closed without merging**